src/V3Os.cpp

```diff
@@ -327,3 +332,11 @@ void V3Os::u_sleep(int64_t usec) {
     ::usleep(usec);
 #endif
 }
+
+//######################################################################
+// METHODS (sub command)
+
+int V3Os::system(const char *command) {
+    const int ret = ::system(command);
```

Comment:
Perhaps this should take a const string& input?

Add before run the comment ~ UINFO(1, "Running system: " << command << endl);
And after ~ UINFO(1, "Command status " << ret <<endl);